app/models/access_token.rb

```diff
@@ -59,11 +49,7 @@ def allowed_for(owner)
 
     def visible_for(account)
       select_and_build do |scope|
-        if scope.public?
-          true
-        else
-          account.provider_can_use?("#{scope.value}_api")
-        end
+        account.provider_can_use?("#{scope.value}_api")
```

Comment:
The `public?` method would only return `false` if the scope was "cms". Only then we'd check for `account.provider_can_use?`.
By removing the "non_public_scopes", this `visible_for` method should be returning `true` all the time, right?
IF that's the case, maybe we could remove it as well.

Reply:
You're right, I misunderstood this method. Removed!